### DISCUSSION TITLE
Ducky Support for Flipper Zero

### DISCUSSION THREAD

**Discussion by jayden-hobbs** [microsoft/vscode-discussions#2432] [Extension Development QnA] [answered]
Hi there!
I'm quite new to coding and was wondering if theres a way for vscode to recognise ducky script? Is it a built in language or do i need to install extension packs to help with it?
Thanks

**Comment by alefragnani** [comment, marked as answer]
Hi @jayden-hobbs ,

I don't know ducky scripts, so I would suggest you to open a file and see if VS Code recognizes it syntax and/or displays anything different than `Plain text` in the status bar. If not, there is no built in support.

If there is no built in support, I would suggest you to search for them in the Extensions Viewlet, using `Extensions: Install Extensions` command. I made a quick search and it seems yes, it does exists. So you could take a look if those fit your needs.

If you had questions about any of the extensions, the best place to look for info would be the extension's repo/page, with their author/team, or Stack Overflow. This discussion repo is intended for extension development only for now.

Hope this helps
